xml/System.Data/DataReaderExtensions.xml

```diff
@@ -389,13 +427,30 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="reader">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="T">The type of the value to be returned.</typeparam>
+        <param name="reader">The <see cref="T:System.Data.Common.DbDataReader" /> to get the column value from.</param>
+        <param name="name">The name of the column.</param>
+        <param name="cancellationToken">An optional <see cref="T:System.Threading.CancellationToken" /> to cancel the asynchronous operation.</param>
+        <summary>Asynchronously gets the value of the specified column as the requestsed type.</summary>
```

Comment:
"requestsed"

Reply:
Is there a better way to refer to the T generic argument of the method?

Reply:
xmldoc supports `<typeparamref name="T"/>`, but I don't know if the website supports it.